src/domain/cache/index.ts

```diff
@@ -3,6 +3,7 @@ export * from "./errors"
 export const CacheKeys = {
   CurrentPrice: "price:current",
   PriceHistory: "price:history",
+  LastOnChainTransactions: "bitcoin:lastOnChainTxs",
```

Comment:
on caveat here is that we call `lastIncomingTransactions` with different depth parameter. I think this cache key here doesn't convey this insights.

maybe cache key could incorporate this somehow?